ceph_deploy/mds.py

```diff
@@ -138,6 +138,11 @@ def mds_create(args):
                 distro.release,
                 distro.codename
             )
+            if distro.normalized_name == 'centos':
```

Comment:
Ungh, it looks like I pushed this by mistake. It was `'redhat'` but I was playing around in CentOS.

The port to modularize this will need to be a separate ticket. It is not hard, but it would be too many changes for this right now.

You raise a good point though and to enforce this rule so that it applies to RHCS it will need to default to not installing it *unless* specified with `--install-mds`. If you agree with that approach I will make the changes and re-push